validator/laichikok-upgrade/README.md

```diff
@@ -0,0 +1,198 @@
+
+## Changelog
+
+- Support new `like` account prefix. The legacy `cosmos` prefix is still supported.
+- Update Cosmos SDK to 0.44.6, ibc-go to 2.1.0.
+
+## Timeline
+
+|Event           |Time                     |
+|----------------|-------------------------|
+|Testnet Proposal|W1 April 2022            |
+|Testnet Deploy  |W2 April 2022, height TBC|
+|Mainnet Proposal|W2 April 2022            |
+|Mainnet Deploy  |W3 April 2022, height TBC|
+
+## Software Versioning Changes
+
+After `fotan-1.1`, releases will be tagged with semantic versioning to facilitate the usage of CI/CD pipelines, e.g. `v1.2.3`. Pre-release versions will contain suffixes such as `-rc1`, `-beta2`, `-alpha3`.
+
+Below is a list of recent stable releases:
+
+- `fotan-1.1`: Current mainnet version
+- `v1.2.0`: First version to support cosmovisor, binary-compatible with mainnet
+- `v2.0.0`: LaiChiKok upgrade
+
+## System Requirements During Upgrade
+
+The system resources required during the upgrade will be higher than normal operation for data migration tasks.
+
+In particular, around 12-20GB memory will be used, and data files will be duplicated for backup.
+
+After the upgrade is applied, the resources requirement is same as previously suggested in our [Node Setup Guide](../likecoin-chain-node/setup-a-node.md).
+
+### Recommended
+
+- 4 CPU Cores
+- 32GB RAM
+- 500GB+ SSD
+
+### Minimum
+
+- 2 CPU Cores
+- 16GB RAM + 16GB Swap
+- 500GB+ SSD
+
+Refer to [this third party guide](https://www.digitalocean.com/community/tutorials/how-to-add-swap-space-on-ubuntu-20-04) for steps to add swap space.
+
+Operators who could not arrange a capable machine might opt to temporarily stop validating until the network has upgraded, then catch up to trusted nodes using state sync.
+
+## Workflow
+
+1. `v2.0.0` is tagged on repo and binary builds are released on github via CI
+
+2. Upgrade proposal is raised on-chain, which specifies the following parameters:
+
+- Upgrade Name: `v2.0.0`
+- Upgrade Height: (TBC)
+- Upgrade Info:
+
+  ```json
+  {
+    "binaries": {
+      "linux/amd64": "https://github.com/likecoin/likecoin-chain/releases/download/v2.0.0/likecoin-chain_2.0.0_Linux_x86_64.tar.gz",
+      "linux/arm64": "https://github.com/likecoin/likecoin-chain/releases/download/v2.0.0/likecoin-chain_2.0.0_Linux_arm64.tar.gz",
+      "darwin/amd64": "https://github.com/likecoin/likecoin-chain/releases/download/v2.0.0/likecoin-chain_2.0.0_Darwin_x86_64.tar.gz",
+      "darwin/arm64": "https://github.com/likecoin/likecoin-chain/releases/download/v2.0.0/likecoin-chain_2.0.0_Darwin_arm64.tar.gz"
+    }
+  }
+  ```
+
+3. After the proposal is approved, operators should place the `v2.0.0` binary at the Cosmovisor upgrades folder BEFORE the upgrade time. Alternatively, operators can configure cosmovisor to download binaries automatically.
+
+4. Cosmovisor will automatically switch binaries at the scheduled block height. Operators will not be required to run migrations or restart chain manually.
```

Comment:
However, it is recommend the operator present during the upgrade time in case of any unpredictable event.

Reply:
added